part-2/2.3_deploying_webservers.md

```diff
@@ -20,18 +20,22 @@ These two instances will be our two web servers
 3. Click *Next: Configure Instance Details*
```

Comment:
On line 12 there's an example of the old diagram

Reply:
Updated!